data/json/recipes/other/parts_construction.json

```diff
@@ -13,6 +13,39 @@
     "proficiencies": [ { "proficiency": "prof_metalworking" }, { "proficiency": "prof_blacksmithing" } ],
     "tools": [ [ [ "hotcut_any", 1, "LIST" ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "crude_bronze_nail",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "charges": 30,
+    "time": "4 h",
+    "autolearn": true,
+    "using": [ [ "bronze_tiny", 1 ] ],
+    "qualities": [ { "id": "ANVIL", "level": 3 }, { "id": "HAMMER", "level": 2 } ],
+    "tools": [ [ [ "forge", 20 ], [ "oxy_torch", 20 ] ] ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" }, { "proficiency": "prof_redsmithing" } ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "bronze_nail",
+    "id_suffix": "simple",
+    "charges": 6,
```

Comment:
Still lots of mass loss here

Reply:
How would you solve it then? It seems unlikely a crude forging of nails from 300g of bronzes would result in 300g of nails, there would be many tiny scraps of molten bronze left behind. If the preference is to have nearer 300g in equals 300g out I'll just change it.

Reply:
> How would you solve it then? It seems unlikely a crude forging of nails from 300g of bronzes would result in 300g of nails, there would be many tiny scraps of molten bronze left behind. If the preference is to have nearer 300g in equals 300g out I'll just change it.

You are losing 276g/300g of bronze here. This is not "leaving tiny scraps of molten bronze behind".

Reply:
... so your suggestion is?

I can only see three options:

1. Make 100s of nails (not very practical or realistic)
2. Have huge losses
3. Introduce a smaller unit of bronze similar to copper 

What would you be happy with?
Do you have any helpful ideas?

Reply:
> What would you be happy with?

More than 6 nails out of a 300g chunk of metal. 6 nails is ridiculously low.

Reply:
It's 30 not 6.

Reply:
> It's 30 not 6.

You are looking at a different recipe that I am reviewing. You've added two recipes in this PR, and only changed one when I asked you to change them.

Reply:
Updated.